app/src/main/java/edu/ucandroid/weathernotice/ui/main/MainFragment.kt

```diff
@@ -106,10 +106,9 @@ class MainFragment : Fragment() {
 
     fun updateAddressUI(location: Location) {
 
-        var geocoder: Geocoder
         var addressList = ArrayList<Address>()
 
-        geocoder = Geocoder(activity!!.applicationContext, Locale.getDefault())
+        var geocoder: Geocoder = Geocoder(activity!!.applicationContext, Locale.getDefault())
```

Comment:
OK I understand.  I'll have review this one with the other since this. I think there is something in the documentation, but will review it. Thank you

Reply:
I believe I found what you are referring to is in [https://kotlinlang.org/docs/coding-conventions.html#source-file-names](https://kotlinlang.org/docs/coding-conventions.html#source-file-names). I am not sure which way is would be considered best practice here. Personally, I think that the change is much more readable.

Reply:
Its not worry I seen other documentation outside of that that is a little different so  am a have get the the team and we go from there, if nothing else probably will take change .